sys/xtimer/xtimer.c

```diff
@@ -128,47 +128,44 @@ void _xtimer_set_msg64(xtimer_t *timer, uint64_t offset, msg_t *msg, kernel_pid_
     _xtimer_set64(timer, offset, offset >> 32);
 }
```

Comment:
Is:
`_callback_msg`
`_setup_msg`
`_xtimer_set_msg`
`_xtimer_set_msg64`
still needed?